sdk/eventhub/event-hubs/tslint.json

```diff
@@ -48,28 +42,17 @@
     "eofline": true,
     "indent": [true, "spaces"],
     "no-trailing-whitespace": true,
-    "arrow-parens": true,
+    "arrow-parens": [true, "ban-single-arg-parens"],
```

Comment:
Only change in this file that matters. Banning the single arg parens for arrow functions since it's conflicting with prettier's default setting, and I am not a big fan of `(x) => {}`.  Though I could change prettier setting too if people prefer.